### PR TITLE
feat(DB/SAI): murloc fear when Mmmrrrggglll die

### PR DESCRIPTION
**Pull request by Helias**
Porting https://github.com/TrinityCore/TrinityCore/issues/20703

## Changes Proposed:
-  improved SAI for Mmmrrrggglll, now the Grimscale Foragers and Grimscale Murlocs NPCs goes in fear when Mmmrrrggglll die

## Issues Addressed:
- Closes https://github.com/chromiecraft/chromiecraft/issues/23

## Tests Performed:
- killed Mmmrrrggglll, verified the fear of the murlocs

## How to Test the Changes:
`.go cr id 15937`
target the NPC
`.die`
when 15937 die the other murlocs should go in fear

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Review by FrancescoBorzi — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1612115546205486724.sql [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+-- Redo script
+-- Mmmrrrggglll SAI
+SET @ENTRY := 15937;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=@ENTRY AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`,`source_type`,`id`,`link`,`event_type`,`event_phase_mask`,`event_chance`,`event_flags`,`event_param1`,`event_param2`,`event_param3`,`event_param4`,`action_type`,`action_param1`,`action_param2`,`action_param3`,`action_param4`,`action_param5`,`action_param6`,`target_type`,`target_param1`,`target_param2`,`target_param3`,`target_x`,`target_y`,`target_z`,`target_o`,`comment`) VALUES
+(@ENTRY,0,0,0,0,0,100,0,0,0,9000,12000,11,12058,0,0,0,0,0,2,0,0,0,0,0,0,0,"Mmmrrrggglll - In Combat - Cast 'Chain Lightning'"),
+(@ENTRY,0,1,0,0,0,100,0,4000,6000,12000,16000,11,21030,0,0,0,0,0,2,0,0,0,0,0,0,0,"Mmmrrrggglll - In Combat - Cast 'Frost Shock'"),
+(@ENTRY,0,2,0,2,0,100,0,0,50,12000,15000,11,12491,1,0,0,0,0,1,0,0,0,0,0,0,0,"Mmmrrrggglll - Between 0-50% Health - Cast 'Healing Wave'"),
+(@ENTRY,0,3,4,6,0,100,0,0,0,0,0,45,0,1,0,0,0,0,9,15668,0,60,0,0,0,0,"Mmmrrrggglll - On Just Died - Set Data 0 1"),
+(@ENTRY,0,4,0,61,0,100,0,0,0,0,0,45,0,1,0,0,0,0,9,15669,0,60,0,0,0,0,"Mmmrrrggglll - On Just Died - Set Data 0 1");
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1612115546205486724.sql:30]: newline

**Review by Kitzunu — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1612115546205486724.sql (hunk 1 above)
Comment (Helias) [reply data/sql/updates/pending_db_world/rev_1612115546205486724.sql]: done, thanks

**Comment by pklloveyou** [comment]
Tested

**Comment by FrancescoBorzi** [comment]
@pklloveyou thank you for all your help with testing, much appreciated <3

**Pull request merged**